plugins/slicer/MONAILabel/MONAILabelLib/client.py

```diff
@@ -164,14 +164,13 @@ def download_label(self, label_id, tag):
         selector = "/datastore/label?label={}&tag={}".format(
             MONAILabelUtils.urllib_quote_plus(label_id), MONAILabelUtils.urllib_quote_plus(tag)
         )
-        status, response, _ = MONAILabelUtils.http_method("GET", self._server_url, selector)
+        status, response, _, headers = MONAILabelUtils.http_method("GET", self._server_url, selector)
         if status != 200:
             raise MONAILabelException(
                 MONAILabelError.SERVER_ERROR, f"Status: {status}; Response: {response}", status, response
             )
 
-        content_disposition = response.getheader("content-disposition")
-        file_name = re.findall('filename="(.+)"', content_disposition)[0]
+        file_name = re.findall('filename="(.+)"', headers["content-disposition"])[0]
```

Comment:
this is list of tuples.. it's something like [(header_name, header_value]+

Reply:
Verified.